src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DottedLineDrawerDriverAdapter.java

```diff
@@ -0,0 +1,37 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class DottedLineDrawerDriverAdapter extends DrawPanelController implements Job2dDriver{
+    private int startX = 0, startY = 0;
+
+    private DrawPanelController drawPanelController;
+    public DottedLineDrawerDriverAdapter(DrawPanelController drawPanelController) {
+        this.drawPanelController=drawPanelController;
+
+
+    }
+
+    @Override
+    public void setPosition(int x, int y) {
+        this.startX = x;
+        this.startY = y;
+    }
+
+    @Override
+    public void operateTo(int x, int y) {
+        ILine line = LineFactory.getDottedLine();
```

Comment:
Hardcoded line type. What about 3.3.2 test?
ILine instance should be configured using Dependency Injection (i.e., field and constructor/setter)